utils/archiver/db_scripts.md

```diff
@@ -0,0 +1,39 @@
+# SQL statements to verify the archiver work and check DB about deprovisioned instances
+
+## DB statistics which should be run before the archiver is started
+
+1. Number of all operations:
```

Comment:
If the sequence is not important here, please change the ordered lists to unordered ones. If it matters, just ignore this comment.